README.md

```diff
@@ -20,11 +20,15 @@ GraphQL tooling & runtime support needed for MS Teams and beyond
 
 - [@graphitation/apollo-mock-client](./packages/apollo-mock-client): An Apollo Client that allows mocking of payloads in response to operations, rather than having to provide them all upfront.
 - [@graphitation/apollo-react-relay-duct-tape](./packages/apollo-react-relay-duct-tape): A compatibility wrapper that provides the react-relay API on top of Apollo Client.
+- [@graphitation/graphitation-cli](./packages/graphitation-cli): Graphitation CLI TODO.
+- [@graphitation/graphql-codegen](./packages/graphql-codegen): GraphQL codegen TODO.
```

Comment:
I think we need another name for this package. Maybe just `@graphitation/codegen`. @alloy @freiksenet WDYT?

Reply:
`ts-codegen` I'd say.